### ISSUE TITLE
Expression rigging not working as expected

### ISSUE THREAD

**Issue by mjvanderboon** [zhuhao-nju/facescape#70]
Hi, 

Thank you for the great work. I am trying to transfer the expression from a source actor to a target actor, however the expression generated for target actor is very minimal. I have attached some pictures to show what the issue is. You can see that the expression generated for the target actor is quite minimal.

Is this due to the limits of the expressiveness of the model? Or should the blendshapes be approached differently?

Actor image
![chan](https://user-images.githubusercontent.com/14217313/143062434-a48024c8-a890-4090-9885-6a446fa97524.jpg)

Expression image
![expression](https://user-images.githubusercontent.com/14217313/143062409-96d4e14c-60aa-4567-aedd-e1a45b7a3998.png)

Actor with inferred expression
![inferred_from_expression](https://user-images.githubusercontent.com/14217313/143062419-b80a1a91-81ec-4cb4-a23d-2211f6136297.png)

**Comment by zhuhao-nju** [comment]
Hi @mjvanderboon ,

We just fixed a bug in this [commit](https://github.com/zhuhao-nju/facescape/commit/a2cf2509dd3f1a1bf50ebe7a4b6a1bc5db25a827) that may cause this problem.  In the code with bug, the result expression vectors of fitting are nearly all zeros. After updating facescape_bm.py, the fitting of expressions works.

I tried to transfer the result expression vector to another face after the bug is fixed, and find that some expressions are good (like mouth-open) but other expressions are bad (like mouth-left). I think it is because of the limited capability of the landmark-based fitting.  In our supplementary video, the face is driven by the expression vector captured by FaceShift that is much more effective.
